src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -2980,16 +2980,57 @@ static WellKnownMembers()
                  (byte)WellKnownType.ExtSentinel, (byte)(WellKnownType.Microsoft_VisualBasic_Conversion - WellKnownType.ExtSentinel),               // DeclaringTypeId
                 0,                                                                                                                                  // Arity
                     1,                                                                                                                              // Method Signature
-                    (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Single,                                    // Return type
-                    (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Single,                                    // Number As System.Single
+                    (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Single, // Return type
+                    (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Single, // Number As System.Single
 
                 // Microsoft.VisualBasic.Conversion.Fix(Number As System.Double)
                 (byte)(MemberFlags.Method | MemberFlags.Static),                                                                                    // Flags
                  (byte)WellKnownType.ExtSentinel, (byte)(WellKnownType.Microsoft_VisualBasic_Conversion - WellKnownType.ExtSentinel),               // DeclaringTypeId
                 0,                                                                                                                                  // Arity
                     1,                                                                                                                              // Method Signature
-                    (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Double,                                    // Return type
-                    (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Double,                                    // Number As System.Double
+                    (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Double, // Return type
+                    (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Double, // Number As System.Double
+
+                 // Microsoft.VisualBasic.Conversion.Int(Number As System.Single)
+                (byte)(MemberFlags.Method | MemberFlags.Static),                                                                                    // Flags
+                 (byte)WellKnownType.ExtSentinel, (byte)(WellKnownType.Microsoft_VisualBasic_Conversion - WellKnownType.ExtSentinel),               // DeclaringTypeId
```

Comment:
Nit: the indentation on the "DeclaringTypeId" line is off by one (including the two from the original PR).